AUR build fails due to missing `write-file-atomic` git commit

* Which version of openWMail are you using? [e.g. 1.3.1]

I am attempting to build openWMail tag v1.0.0.

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]

I am using Arch Linux, trying to build openWMail using [the `PKGBUILD`](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=openwmail) from the Arch User Repository](https://aur.archlinux.org/packages/openwmail/).

* Further environment info:

```
$ uname -a
Linux ahost 5.2.5-arch1-1-ARCH #1 SMP PREEMPT Wed Jul 31 08:30:34 UTC 2019 x86_64 GNU/Linu
```

```
$ makepkg --version
makepkg (pacman) 5.1.3
Copyright (c) 2006-2018 Pacman Development Team <pacman-dev@archlinux.org>
Copyright (C) 2002-2006 Judd Vinet <jvinet@zeroflux.org>.

This is free software; see the source for copying conditions.
There is NO WARRANTY, to the extent permitted by law.
```

```
$ bash -c 'source PKGBUILD; for exe in "${makedepends[@]}" "${depends[@]}"; do printf "# %s\\n" "$exe" "$("$exe" --version 2>&1)"; done'
# git
# git version 2.22.0
# npm
# 6.10.2
# python2
# Python 2.7.16
# electron
# v5.0.8
```

* How to reproduce:

```
$ makepkg -cis
# <snip a bunch of output from npm>
npm ERR! code 128
npm ERR! Command failed: git checkout 5be21f07e36664e47a7e342c26d8ee93424b5f42
npm ERR! fatal: reference is not a tree: 5be21f07e36664e47a7e342c26d8ee93424b5f4
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/auser/.npm/_logs/2019-08-07T14_21_16_440Z-debug.log
```

* Notes:

Looks like the commit referenced in the `npm` error message pertains to the `write-file-atomic` dependency.  The commit is [still visible on GitHub](https://github.com/geekgonecrazy/write-file-atomic/commit/5be21f07e36664e47a7e342c26d8ee93424b5f42), but isn't reachable using the `git` client (or, presumably, by whatever Git interface `npm` uses under the hood):

```
$ git remote -v
origin  https://github.com/geekgonecrazy/write-file-atomic (fetch
origin  https://github.com/geekgonecrazy/write-file-atomic (push)
$ git rev-list --remotes --all | grep -Fq 5be21f07e36664e47a7e342c26d8ee93424b5f42 || echo ':(
:(
```

More than happy to offer any further info that might be helpful.  Thanks in advance for any assistance you can offer!

At least one other Arch Linux / AUR user has [hit this same issue](https://aur.archlinux.org/packages/openwmail/#comment-703250):

> When installing with pkgbuilder, I get this error:
>
> ```
> npm ERR! Command failed: git checkout 5be21f07e36664e47a7e342c26d8ee93424b5f42
> npm ERR! fatal: reference is not a tree: 5be21f07e36664e47a7e342c26d8ee93424b5f42
> npm ERR!
> ```

--- a/src/app/package.json
+++ b/src/app/package.json
@@ -24,7 +24,7 @@
     "uuid": "3.0.1",
     "windows-shortcuts": "openWMail/windows-shortcuts#0.1.4",
     "wmail-spellchecker": "openWMail/wmail-spellchecker#1.0.5",
-    "write-file-atomic": "git://github.com/geekgonecrazy/write-file-atomic.git#5be21f07e36664e47a7e342c26d8ee93424b5f42",
+    "write-file-atomic": "^3.0.0",
     "yargs": "6.6.0"
   },
   "devDependencies": {

